src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/viewmodels/IndexViewModel.kt

```diff
@@ -0,0 +1,13 @@
+package org.vaccineimpact.orderlyweb.viewmodels
+
+import org.vaccineimpact.orderlyweb.ActionContext
+import org.vaccineimpact.orderlyweb.models.Report
+
+open class IndexViewModel(context: ActionContext,
+                          open val reports: List<Report>) : AppViewModel(context, this.breadCrumb)
+{
+    companion object
+    {
+        val breadCrumb = BreadCrumb("Main menu", "/")
```

Comment:
Is the home page really a "Main menu" if it's only got one sort of thing on it? Would 'All reports' be more accurate? Maybe fine if everyone's happy with it in the reportle!

Reply:
I'll leave for now as this is how it is in the reportle. Could change as part of https://vimc.myjetbrains.com/youtrack/issue/mrc-263 if needed.